README.md

```diff
@@ -42,12 +42,15 @@ Future features include managing/creating email campaigns and ticket exchanges.
 3. Create mkcert certificates.
    1. Navigate to `<path/to/WonderTix/server>` and `<path/to/WonderTix/client>`.
    2. Run `mkcert -install` to install the local certificate authority.
-   3. Run `mkcert localhost` to create a certificate.   
+   3. Run `mkcert localhost` to create a certificate.
+   4. In the server directory, run the following command:
```

Comment:
It would be nice if this step briefly described what the command is and why you need to do it

Reply:
Got it; added a couple sentences of explanation.